### PR TITLE
Add group

### PR DESCRIPTION
**Pull request by Bathande**

### CODE REVIEW

File: skietbaan_fe/src/actions/types.js.orig [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export const ADD_BOOKMARK = 'ADD_BOOKMARK';
+export const DELETE_BOOKMARK = 'DELETE_BOOKMARK';
+export const FETCH_POSTS = 'FETCH_POSTS';
+export const NEW_POST = 'NEW_POST';
+export const NEW_COMP = 'NEW_COMP';
+export const FETCH_COMP = 'NEW_COMP';
```
Comment (FredJEhlers) [inline skietbaan_fe/src/actions/types.js.orig]: remove `.orig` files

File: skietbaan_fe/src/components/AddMembersGroup.js [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    })
+      .then(function(response) {})
+      .then(function(data) {})
+      .catch(function(data) {});
+  }
+  toggleHighlight =(name,event)  => {
+
+    if (this.state.posts[event].highlighted === true) {
+      this.state.posts[event].highlighted = false;
+
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/AddMembersGroup.js]: empy lines and indentation also look at the else

File: skietbaan_fe/src/components/EditGroup.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+        newArray.push(this.state.posts[i]);
+      }
+      delete this.state.posts[i].highlighted;
+      delete this.state.posts[i].id;
+    }
+    let request = {
+      GroupIds:this.props.id,
+      users: this.state.newArray
+    };
+    fetch("https://api.skietbaan.retrotest.co.za/api/groups/deleteMember/", {
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/EditGroup.js]: BASE_URL please

File: skietbaan_fe/src/components/EditGroup.js [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+      method: "DELETE",
+      headers: {
+        Accept: "application/json",
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify(request)
+    })
+      .then(function(response) {})
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/EditGroup.js]: I dont understand why you have two empty `.then()`

File: skietbaan_fe/src/components/Groups.js [hunk 5]
```diff
@@ -1,117 +1,180 @@
...
+    };
+    this.toggleHighlight = this.toggleHighlight.bind(this);
+    this.handleOnClick = this.handleOnClick.bind(this);
+    this.onBack = this.onBack.bind(this);
+    this.onChange = this.onChange.bind(this);
+    this.selectall = this.selectall.bind(this);
+  }
   componentWillMount() {
-    fetch('http://localhost:63474/api/user')
+    fetch("https://api.skietbaan.retrotest.co.za/api/user")
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/Groups.js]: Make sure to use BASE_URL in all casses

File: skietbaan_fe/src/components/Groups.js [hunk 6]
```diff
@@ -1,117 +1,180 @@
...
+    fetch("https://api.skietbaan.retrotest.co.za/api/groups/add", {
+      method: "post",
+      headers: {
+        Accept: "application/json",
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify(newArray)
+    })
+      .then(function(response) {})
+      .then(function(data) {})
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/Groups.js]: same comment as above about the `.then`

File: skietbaan_fe/src/components/ViewGroups.css [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+.check {
+  color: #101727;
+  display: flex;
+}
+
+.boxes {
+  float: right;
+  margin-top: 20px;
+}
+.blabe{
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/ViewGroups.css]: empty space between classes and check indentation

Comment (FredJEhlers) [reply skietbaan_fe/src/components/ViewGroups.css]: empty line EOF

File: skietbaan_fe/src/components/add.css [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+  margin-left: auto;
+  margin-left: auto;
+  text-align: center;
+
+}
+/* .center_label {
+  width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  width: 8em;
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/add.css]: GC

File: skietbaan_fe/src/components/groups.css [hunk 9]
```diff
@@ -1,85 +1,279 @@
...
+}
+.updatess{
+  width: 150px;
+  height: 39px;
+  border-radius: 40px;
+  width: 96px;
+  height: 42px;
+
+}
+.botname{
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/groups.css]: check all the css files about my css comments

File: skietbaan_fe/src/components/EditGroup.js [hunk 10]
```diff
@@ -8,18 +8,18 @@ class EditGroup extends Component {
...
+      count:0
     };
     this.toggleHighlight = this.toggleHighlight.bind(this);
     this.handleOnClick = this.handleOnClick.bind(this);
     this.onBack = this.onBack.bind(this);
     this.onChange = this.onChange.bind(this);
   }
   componentWillMount() {
-    fetch("http://localhost:63474/api/Groups/edit?id=" + this.props.id)
+    fetch("http://localhost:64444/api/Groups/edit?id=" + this.props.id)
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/EditGroup.js]: No Localhost variables
This makes me also think you missed some base url request please make sure to get all of them

Comment (Bathande) [reply skietbaan_fe/src/components/EditGroup.js]: thanks i have made the changes

**Pull request merged**